vars/folioNamespaceCreate.groovy

```diff
@@ -4,13 +4,16 @@ import org.folio.models.parameters.CreateNamespaceParameters
 import org.folio.rest.GitHubUtility
 import org.folio.rest_v2.Edge
 import org.folio.rest_v2.Main
+import org.folio.utilities.Logger
```

Comment:
This lib's changes are bringing additional overhead in merging Eureka LLB into master, very sad situation.

Reply:
Yeap, this is a pain of long-lived branches